### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#602] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/

小白指南 这里是一个精确到每一步的小白指南: 1. 准备运行环境 (tutorial/env.md) 1. 创建项目 (tutorial/create.md) 1. 项目命令 (tutorial/command.md) 1. 项目内容 (tutorial/content.md) 1. 项目配置 (tutorial/config.md) 1. 项目结构 ...

https://theme-hope.vuejs.press/zh/cookbook/

**Comment by 307523005** [comment]
?
